lib/ActivityStream.jsm

```diff
@@ -239,6 +239,10 @@ const PREFS_CONFIG = new Map([
       });
     },
   }],
+  ["discoverystream.endpoints", {
```

Comment:
This is a separate pref on purpose to make it impossible to change via shield? Or do we want to consider allowing moving this to our primary json blob pref so shield can modify it while doing an experiment?

Again, this might be intended as something we explicitly don't want to allow, and if so, this makes sense.

Reply:
Nod. I believe security wanted a standalone pref easy to check if it's been modified. For our own normandy pref flip studies, do we expect to ever point large amount of users not at the cdn?